civictechprojects/models.py

```diff
@@ -441,13 +445,13 @@ def hydrate_project_volunteer_info(self):
         project_json = self.project.hydrate_to_list_json()
         return merge_dicts(volunteer_json, project_json)
 
-    def update_project_timestamp(self):
-        self.project.save()
+    def update_project_timestamp(self, user):
+        self.project.save(user=user)
```

Comment:
This won't actually update the project timestamp anymore now that project_date_modified doesn't update automatically.  Is it even being used anywhere at this point?

Reply:
Agreed. Removed it.